components/cards/RelaxStepCard.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <div class="RelaxStep">
+    <div class="RelaxStep-heading">
+      <h3 class="RelaxStep-title">
+        {{ $t('施設別休止要請の緩和ステップ') }}
+      </h3>
+      <div class="RelaxStep-link">
+        <link-to-information-about-roadmap />
+      </div>
+    </div>
+    <div class="RelaxStep-content">
+      <div class="RelaxStep-description">
+        <p>
+          {{
+            $t(
+              '休業要請緩和のステップについては、令和2年5月22日時点の状況を「ステップ0」とし、施設の特性やクラスター発生歴等から判断して、「ステップ3」までの4段階を設定し、段階的に施設の休業要請を緩和します。'
+            )
+          }}
+        </p>
+      </div>
+      <div>
+        <ul class="RelaxStep-steps">
+          <li v-for="i in steps" :key="i" class="RelaxStep-steps">
+            <p v-if="i === RelaxStep.step" class="RelaxStep-steps-on">
+              {{ $t('ステップ {num}', { num: i }) }}
+            </p>
+            <p v-else class="RelaxStep-steps-off">
+              {{ $t('ステップ {num}', { num: i }) }}
+            </p>
+          </li>
+        </ul>
+        <p class="RelaxStep-changed-text">
+          {{
+            $t('{date} ステップ {num}に移行', {
+              date: formattedDayForDisplay,
+              num: RelaxStep.step
+            })
+          }}
+        </p>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import LinkToInformationAboutRoadmap from '@/components/LinkToInformationAboutRoadmap.vue'
+import RelaxStep from '@/data/tokyo_alert.json'
+
+export default Vue.extend({
+  components: {
+    LinkToInformationAboutRoadmap
+  },
+  props: {
+    items: {
+      type: Array,
+      required: true
+    }
+  },
+  data() {
+    return {
+      RelaxStep,
+      steps: [0, 1, 2, 3]
+    }
+  },
+  computed: {
+    formattedDayForDisplay() {
+      return this.$d(new Date(RelaxStep.changed), 'dateWithoutYear')
```

Comment:
ステップの日付は時刻も表示させたいので、 `utils/formatDate.ts` の `convertDatetimeToISO8601Format` を使うといいと思います。

Reply:
```suggestion
      return this.$d(new Date(RelaxStep.changed), 'dateTime')
```
年を表示して大丈夫であればこれで対応できそうです。

Reply:
> 年を表示して大丈夫であればこれで対応できそうです。

はい、大丈夫です！

Reply:
`convertDatetimeToISO8601Format ` だと `2020-06-03T18:30:00` みたいなやつになります
「M月D日（曜日） 午前/午後 H時」形式も定義する必要ありますかね？

Reply:
@MaySoMusician 厳密でなくていいとは思いますが、最低限、曜日は再現したいですね。「午前／午後」まではなくていいと思います。

Reply:
午前／午後は厳しいですが、曜日の再現は可能だったような気がします。先ほど提示したものとは別のアプローチを試してみます。

Reply:
@y-chan 緩和ステップ用にひとつフォーマットを定義しても良いかと思います！
#4696 

![image](https://user-images.githubusercontent.com/34566290/83703486-bf919000-a64a-11ea-8eaa-1832dd48b484.png)

Reply:
午前／午後も対応できるかもしれないです

Reply:
#4697 で定義追加PR出してみました（定義名: `dateTimeRelaxationSteps` ）
`this.$d(new Date(RelaxStep.changed), 'dateTimeRelaxationSteps')` という感じで使えるようになります

日本語と英語では語順や区切り文字が異なるので vue-i18nに任せてしまった方がいいかなと思います

Reply:
対応できました！
![image](https://user-images.githubusercontent.com/34832037/83705131-0c776580-a64f-11ea-8d95-487e3aee92c1.png)